.github/workflows/documentation-windows.yml

```diff
@@ -60,21 +60,28 @@ jobs:
         # Avoid annoying warning: "xelatex: major issue: So far, you have not checked for updates as a MiKTeX user."
         mpm --find-updates
         mpm --admin --find-updates
+        # initexmf --enable-installer --update-fndb
+        # initexmf --admin --enable-installer --update-fndb
+        # initexmf --enable-installer --dump-by-name=xelatex --engine=xetex
```

Comment:
Should these lines be deleted now?

Reply:
They could, but they don't hurt either, and I'm pretty sure I'll find them helpful next time there is a problem :D